solvation_analysis/tests/test_solvation_analysis.py

```diff
@@ -77,28 +85,76 @@ def u_real_named(u_real):
 
 @pytest.fixture
 def atom_groups(u_real):
-    li_atoms = u_real.atoms.select
-    pf6_atoms = u_real.atoms.select
-    bn_atoms = u_real.atoms.select
-    fec_atoms = u_real.atoms.select_atoms()
+    li_atoms = u_real.atoms.select_atoms("type 22")
+    pf6_atoms = u_real.atoms.select_atoms("byres type 20")
+    bn_atoms = u_real.atoms.select_atoms("byres type 5")
+    fec_atoms = u_real.atoms.select_atoms("byres type 21")
     atom_groups = {"li": li_atoms, "pf6": pf6_atoms, "bn": bn_atoms, "fec": fec_atoms}
-
-
-def test_solvation_analysis_imported():
-    """Sample test, will always pass so long as import statement worked"""
-    assert "solvation_analysis" in sys.modules
+    return atom_groups
 
 
 def test_get_atom_group(u_real_named):
-    return
-
-
-def test_get_closest_n_mol():
-    return
-
-
-def test_get_radial_shell():
-    return
+    u = u_real_named
+    res_group = u.residues[1:5]
+    atom_group = u.atoms[1:5]
+    res = u.residues[1]
+    atom = u.atoms[1]
+    groups = [res_group, atom_group, res, atom]
+    for group in groups:
+        assert isinstance(get_atom_group(u, group), mda.core.groups.AtomGroup)
+
+
+def test_get_closest_n_mol_grid(u_grid_1):
+    u = u_grid_1
+    test_atoms = u.atoms[[0, 5, 10, 16, 42]]
+    for test_atom in test_atoms:
+        atoms = get_closest_n_mol(u, test_atom)
```

Comment:
Would you be able to put the default arguments here in full? This test relies heavily on the default signature of `n_mol=5`. Including it will make the test much easier to read.